### PR TITLE
Upgrading libs

### PR DESCRIPTION
**Pull request by juanpedromoreno**
This PR upgrades dependencies.

Closes #1083
Closes #1082
Closes #1078
Closes #1075
Closes #1073
Closes #1069
Closes #1048
Closes #1019
Closes #975

### CODE REVIEW

File: modules/tests/src/test/scala/higherkindness/mu/rpc/kafka/ServiceSpec.scala [hunk 1]
```diff
@@ -102,25 +102,26 @@ class ServiceSpec extends AnyFunSuite with Matchers with OneInstancePerTest with
           topicName <- "topic".pure[IO]
           create    <- client.createTopic(CreateTopicRequest(topicName, 2, 1)).attempt
           _         <- IO(assert(create.isRight))
-          describe  <- client.describeTopics(DescribeTopicsRequest(List(topicName))).attempt
```
Comment (juanpedromoreno) [inline modules/tests/src/test/scala/higherkindness/mu/rpc/kafka/ServiceSpec.scala:105]: ℹ️  I had to comment on some of the assertions in this file, as they fail after upgrading the `embedded-kafka `version. It'll need to be fixed in a separate PR.

**Review by franciscodr — APPROVED** [review]

File: modules/tests/src/test/scala/higherkindness/mu/rpc/kafka/ServiceSpec.scala (hunk 1 above)
Comment (BenFradet) [reply modules/tests/src/test/scala/higherkindness/mu/rpc/kafka/ServiceSpec.scala:105]: I think we can remove that module altogether, no?
